package.json

```diff
@@ -12,6 +12,7 @@
     "chai": "^3.4.1",
     "cucumber": "^0.9.2",
     "david": "^7.0.0",
+    "debug": "^2.2.0",
```

Comment:
While that's true, the only reason it works locally for you is one of your dependencies happen to require debug. If that dependency stops requiring debug this will break locally for you. You should explicitly define any dependencies you have in package.json